salt/modules/zypper.py

```diff
@@ -21,6 +21,10 @@
 import os
 import time
 import datetime
+try:
+    from salt.utils.versions import LooseVersion
+except ImportError:
+    LooseVersion = None
```

Comment:
I don't understand this. You seem to be trying to trade an exception in the module load for an exception when you try to call a ``None`` later on.

There really isn't a way that this import should ever fail either, since ``salt.utils.versions.LooseVersion`` is based on ``distutils.version.LooseVersion`` from the Python stdlib.